controllers/servers/host_definer/watcher/exceptions.py

```diff
@@ -0,0 +1,52 @@
+class WatcherException(Exception):
+    def __str__(self, *args, **kwargs):
+        return self.message
+
+
+class SecretDoesNotExist(WatcherException):
+    def __init__(self, secret_name, secret_namespace):
+        super().__init__()
+        self.message = 'Secret {} in namespace {} does not exist'.format(
```

Comment:
do we want `messages.py` for it?
@oriyarde what do you think?

Reply:
sure

Reply:
as they are not common please put the definer messages seperate in the host_definer package